README.md

```diff
@@ -93,7 +95,7 @@ ignored_user_logins = [
 #
 # If you modify this regex, make sure to match the content with a capture
 # group named "summary".
-pr_summary_regex = "^```release-note\\s*(?P<summary>[\\s\\S]*?\\w[\\s\\S]*?)\\s*^```"
+pr_summary_regex = "^```release-note\\s*((?P<label>[^:]*):)?(?P<summary>[\\s\\S]*?\\w[\\s\\S]*?)\\s*^```"
```

Comment:
As a start I'm proposing the following syntax (ignore escaping `\`):

```
\```release-note
Enhancement: Add new `mask` parameter to `foo` to allow masking of values.
\```

\```release-note
Bug fix: Fix endless loop in `foo` if NaN is passed as input. 
\```
```
These two will be translated into two items, one in the "Enhancements" and one in the "Bug fixes" section. If a preceding `Label: ` isn't given, the labels of the PR are used.

Reply:
What I was trying to say is that, unfortunately, contributors cannot see the fenced code block without editing the PR description. But, with the release notes heading you added, that's not such a big problem.

I still think this might be a bit brittle, since colons are so commonly used in descriptions of all sorts. However, if it's limited to "SINGLE-WORD: stuff" it seems less likely. So, let's give it a shot.

Reply:
Hmm, it's currently not limited to single words infront of the colon. What do you think about a compromise that uses two colons?

```
\```release-note
Enhancement:: Add new `mask` parameter to `foo` to allow masking of values.
\```

\```release-note
Bug fix :: Fix endless loop in `foo` if NaN is passed as input. 
\```
```
Alternatively we could just move the `{label="release-note"}` into the next line so that it's visible. E.g. 
```
{label="release-note"} Some note.
```
What I like about this approach is that the `label=` part makes it more expressive and also that the position doesn't matter.